File: elementary/correct-sentence.py
#!/usr/bin/env python3

def correct_sentence(text: str) -> str:
    return text[0].upper()+text[1:]+("" if text[-1] == "." else ".")

if __name__ == '__main__':
    print(correct_sentence("hello, world"))
    assert correct_sentence("hello, world") == "Hello, world."
    assert correct_sentence("kis mokus felmaszott a fara.") == "Kis mokus felmaszott a fara."

File: elementary/correct-sentence.rb
#!/usr/bin/env ruby
#
def correct_sentence(text)
    return (text.end_with?('.') ? text : text+".").capitalize
end

if __FILE__ == $0
    puts(correct_sentence("hello, world"))
    raise unless correct_sentence("hello, world") == "Hello, world."
    raise unless correct_sentence("kis mokus felmaszott a fara.") == "Kis mokus felmaszott a fara."
end
